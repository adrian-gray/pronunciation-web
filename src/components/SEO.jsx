import React from 'react'
import { Helmet } from 'react-helmet'

import meta from './../data/meta'

const SEO = (props) => {
  const content = Object.assign({}, meta['default'], meta[props.meta])
  const { canonical, description, title, imageFB, imageTW } = content

  return (
    <Helmet>
      <title>{ title }</title>
      <meta name='description' content={description} />

      <meta property='og:url' content={canonical} />
      <meta property='og:type' content='website' />
      <meta property='og:image' content={imageFB} />
      <meta property='og:title' content={title} />
      <meta property='og:image:secure_url' content={imageFB} />
      <meta property='og:description' content={description} />

      <meta name='twitter:site' content={canonical} />
      <meta name='twitter:title' content={title} />
      <meta name='twitter:description' content='' />
      <meta name='twitter:image' content={imageTW} />
    </Helmet>
  )
}

export default SEO
