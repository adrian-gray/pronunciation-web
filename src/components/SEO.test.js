import React from "react";
import { shallow } from "enzyme";
import { render, cleanup } from "@testing-library/react";

import SEO from "./SEO";

afterEach(cleanup);

test("SEO", async () => {
  const props = {};

  const container = shallow(<SEO {...props} />);

  expect(container.firstChild).toMatchSnapshot();
});
