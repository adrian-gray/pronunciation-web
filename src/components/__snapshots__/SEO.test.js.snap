// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO 1`] = `undefined`;
